### PR TITLE
Implement enforcement and self-auditing content

### LINKED ISSUE #526: HIPAA page may want to mention testing / enforcement / self-auditing
Per some feedback from customers, something our HIPAA landing page should mention is how we test for compliance, enforce it on an ongoing basis, and "prove" compliance through self-auditing code. In particular, we may want to call out:

1. Terratest to test basic functionality.
2. Some off-the-shelf compliance-checking solution we use to give you a green check or red "X."
3. OPA policies to enforce compliance on an on-going basis. This seems especially popular these days.

### PR DESCRIPTION
**Pull request by zackproser**
Fixes #526

### CODE REVIEW

**Review by brikis98 — COMMENTED** [review]
Thanks Zack!

Two NITs on the new images:

1. You might want to remove the top part of the `aws-security-hub-green.png` image, which says "CIS AWS Foundation Controls."
2. Instead of a screenshot of the OPA home page, which is a bit weird, copy/paste the actual code snippet, and include it in a `code` and `pre` block. That'll make it more readable.

**Comment by zackproser** [comment]
> Thanks Zack!
> 
> Two NITs on the new images:
> 
>     1. You might want to remove the top part of the `aws-security-hub-green.png` image, which says "CIS AWS Foundation Controls."
> 
>     2. Instead of a screenshot of the OPA home page, which is a bit weird, copy/paste the actual code snippet, and include it in a `code` and `pre` block. That'll make it more readable.

Roger, will do!

**Review by ebeneliason — APPROVED** [review]
Looks good to me! One question I have is whether these should perhaps come before step 5, since they are specific features of this offering whereas code updates are more general, but I wouldn't hold up the show on it.

File: pages/landing/hipaa/index.html [hunk 1]
```diff
@@ -52,6 +54,33 @@
...
+      <figure><figcaption>Ensure all images come from a trusted registry</figcaption>
+           <pre>
+            <code class="language-python">
+              package kubernetes.admission
+
+              deny[msg] {
+                  input.request.kind.kind == "Pod"
+                  some i
+                  image := input.request.object.spec.containers[i].image
+                  not startswith(image, "hooli.com/")
```
Comment (yorinasub17) [inline pages/landing/hipaa/index.html:68]: NIT: should we replace this with `gruntwork.io` (this rule is saying all kubernetes container images must source from `hooli.com`, which is an odd example for us to have on our website).

**Review by yorinasub17 — APPROVED** [review]

File: pages/landing/hipaa/index.html (hunk 1 above)
Comment (zackproser) [reply pages/landing/hipaa/index.html]: Yup - fixed in 86869d52

**Review by yorinasub17 — APPROVED** [review]

**Comment by zackproser** [comment]
Thanks for the reviews!

**Pull request merged**